src/ray/common/network_util.cc

```diff
@@ -16,49 +16,170 @@
 
 #include "ray/util/logging.h"
 
+typedef std::pair<std::string, std::string> NicAndIp;
+
+bool StartsWith(std::string s, std::string start) {
+  return s.find(start) == 0;
+}
+
+bool CompNicsAndIps(NicAndIp a, NicAndIp b){
+  return GetNicPriority(a.first) < GetNicPriority(b.first);
+}
+
+int GetNicPriority(std::string nic_name){
+  int priority = PRIORITY_TO_DELETE;  // smaller number, more priority
+
+  if(StartsWith(nic_name, "e")){  // eth0, enp7s0f1, ens160, en0, etc
+    priority = 1;
+  }
+  else if (StartsWith(nic_name, "w")) {  // wlp0s20f3, wlp2s0, etc
+    priority = 10;
+  }
+  else if (StartsWith(nic_name, "br")) {
```

Comment:
How about define an array that includes all prefixes and its priority, then we can traverse it to call StartsWith and get priority, this will make the code simpler, thanks.